prisma/migrations/20240110213938_rename_blockchain/migration.sql

```diff
@@ -0,0 +1,16 @@
+/*
+  Warnings:
+
+  - The values [Optimism_Sequoia] on the enum `Chain` will be removed. If these variants are still used in the database, this will fail.
```

Comment:
```suggestion
  - The values [Optimism_Sepolia] on the enum `Chain` will be removed. If these variants are still used in the database, this will fail.
```